core/tests/protractor.conf.js

```diff
@@ -68,39 +69,59 @@ exports.config = {
     ],
 
     collections: [
-      'protractor/collections.js'
+      'protractor_desktop/collections.js'
```

Comment:
FYI: it actually used to be the case that we could have multiple files per test suite, but this was removed due to an error with chromedriver 2.38 and that's why we started combining e2e tests for different things within the same JS file (which honestly is a bit icky).

That issue might be fixed now (I remember seeing that they were aware of the bug and that there was a bug). If so, you might have more flexibility here in how to handle the grouping of files within test suites. And now that we download a specific version of Chrome, it should be easier to avoid spurious breakages like that one.

So, just an FYI in case it's useful for your grouping of things.